test/hotspot/jtreg/compiler/c2/TestMergeStores.java

```diff
@@ -797,11 +797,10 @@ static Object[] test2RBE(byte[] a, int offset, long v) {
     }
 
     @Test
-    @IR(counts = {IRNode.STORE_B_OF_CLASS, "byte\\\\[int:>=0] \\\\(java/lang/Cloneable,java/io/Serializable\\\\)", "8",
-                  IRNode.STORE_C_OF_CLASS, "byte\\\\[int:>=0] \\\\(java/lang/Cloneable,java/io/Serializable\\\\)", "0",
-                  IRNode.STORE_I_OF_CLASS, "byte\\\\[int:>=0] \\\\(java/lang/Cloneable,java/io/Serializable\\\\)", "0",
-                  IRNode.STORE_L_OF_CLASS, "byte\\\\[int:>=0] \\\\(java/lang/Cloneable,java/io/Serializable\\\\)", "0"},
-        applyIfPlatform = {"little-endian", "true"})
+    @IR(counts = { IRNode.STORE_L_OF_CLASS, "byte\\\\[int:>=0] \\\\(java/lang/Cloneable,java/io/Serializable\\\\)", "1",
+                   IRNode.REVERSE_BYTES_L, "1"},
+        applyIf = {"UseUnalignedAccesses", "true"},
+        applyIfPlatformAnd = {"little-endian", "true", "riscv64", "false"})   // Exclude RISV64 because ReverseBytesI/L are not supported
```

Comment:
Can you please leave in the checks for the other stores (B, C, I)? I would like to have an exact count of each. This is so that we are sure that the others have disappeared as expected, not that some of them linger around and worsen performance.

Reply:
Same in all the cases below